File: app/src/main/java/com/gcode/vastutils/activity/TestActivity.java
package com.gcode.vastutils.activity;

import android.content.Context;
import android.os.Bundle;
import android.util.AttributeSet;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.gcode.vastutils.databinding.ActivityTestBinding;

import java.lang.reflect.Constructor;
import java.util.HashMap;
import java.util.Map;

public class TestActivity extends AppCompatActivity {

    private ActivityTestBinding mBinding;

    private static final Map<String,Constructor<? extends View>> mConstructorMap = new HashMap<>();

    private static final Class<?>[] mConstructorSignature = new Class[]{
            Context.class, AttributeSet.class
    };

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBinding = ActivityTestBinding.inflate(getLayoutInflater());
        setContentView(mBinding.getRoot());
    }

}

File: app/src/main/java/com/gcode/vastutils/activity/TestActivity.kt
package com.gcode.vastutils.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.LinearLayout
import com.gcode.vastutils.R

class TestActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)

        val linear = findViewById<LinearLayout>(R.id.linear)

        // 将Button 加入到LinearLayout 中
        val b1 = Button(this);
        b1.setText("取消");
        linear.addView(b1);

        // 将Button 2 加入到LinearLayout 中
        val b2 = Button(this);
        b2.setText("确定");
        linear. addView ( b2 );

    }
}